apps/server/src/modules/board/controller/dto/element/element-content-update.body.params.ts

```diff
@@ -24,13 +27,13 @@ export class FileContentBody {
 	caption!: string;
 }
 
-export class TextElementContentBody extends ElementContentBody {
-	@ApiProperty({ type: ContentElementType.TEXT })
-	type!: ContentElementType.TEXT;
+export class RichTextElementContentBody extends ElementContentBody {
+	@ApiProperty({ type: ContentElementType.RICH_TEXT })
+	type!: ContentElementType.RICH_TEXT;
 
 	@ValidateNested()
 	@ApiProperty()
```

Comment:
I guess here we are missing sanitization

Reply:
You're right, forgot about that part somehow

Reply:
here in the params, we still don't apply sanitization. 
Looks like you apply it on the response instead of here, so after the data is inserted in db. Is that on purpose?